circle.yml

```diff
@@ -0,0 +1,30 @@
+machine:
```

Comment:
you need to activate CI for this new repo. and make sure it's open source friendly.